eras/shelley/impl/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## 1.12.2.1
```

Comment:
Path version should only be used for non-semantic and not visible to outside users changes. Addition of `submitFailingTxM` is a visible non-breaking change, so minor version needs to be bumped. Moreover, `submitFailingTxM` is used in `cardano-ledger-alonzo`, which means we need to add a lower bound for it: `cardano-ledger-shelley >= 1.12.3`

```suggestion
## 1.12.3.0
```